.bazelrc

```diff
@@ -70,20 +80,7 @@ build:release --config=secure
 build:release --copt -mllvm --copt -inline-threshold=2500
 build:release --linkopt=-flto
 
-build --keep_going
-
-# prevent actions and tests from using the network. anything that needs to
-# opt-out (e.g. a network test) can use `tags=["requires-network"]`.
-build --sandbox_default_allow_network=false
-
-# prevent certain environment variables (e.g. PATH and LD_LIBRARY_PATH) from
-# leaking into the build.
-build --incompatible_strict_action_env
-
-# disabling until we are able to address the issue with GHA checkout into
-# container resulting in a Git repository that doesn't seems to cause errors
-# https://github.com/actions/checkout/issues/363
-#build --workspace_status_command=bazel/workspace_status.sh
```

Comment:
wanna remove workspace_status.sh?

Reply:
Lol I didn't even see that file. Yeah will do.

Reply:
Fixed